backend/smalldata/views.py

```diff
@@ -21,11 +22,11 @@ def perform_create(self, serializer):
         #predict class
         category_name, prob = Classifier.predict_proba(text)
         """
-        # cat, prob = Classifier.predict_proba(text, verbose=True)
+        # cat, prob = clf.predict_proba(text, verbose=True)
         category_name = 'Lob'
-
         category = Category.objects.all().filter(name=category_name)
```

Comment:
@Regexose the error you see is this:
```ValueError at /api/utterances/
Cannot assign "<QuerySet [<Category: Category object (1)>]>": "Utterance.category" must be a "Category" instance```
It says that attribute `category` of the class `Utterance` must a be `Category`-class, but the value you are providing is a `<QuerySet [<Category: Category object (1)>]>`

The reason is that the call `Category.objects.all().filter(name=category_name)` a list of all elements that fulfil the criteria `name=category_name` (to be precise, its not a list but a `QuerySet`). To really get a `Category`-instance, you can simply take the first element of the list, i.e. 
```
category = Category.objects.all().filter(name=category_name)[0]
```

Reply:
ah, that works now

Reply:
@staudamm 
Classifier works too!
zumindest prinzipiell, die Erkennung selbst ist eher suboptimal..